eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -56,7 +56,7 @@ stages:
                     - checkout: self
                     - task: PowerShell@2
                       displayName: Prep template pipeline for release
-                      condition: and(succeeded(),eq(variables['TestPipeline'],'true'))
+                      condition: and(succeeded(),eq('${{ parameters.TestPipeline }}','true'))
```

Comment:
We should define the TestPipeline parameter in this file and set the default to either empty or false.